sklearn/metric_learning/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+The :mod:`sklearn.metric_learning` module includes models attributed to
+Metric Learning field.
```

Comment:
I'm sure you can come up with a better brief summary of the field than this, which conveys little more than the module's name itself ;)